pkg/controller/controller.go

```diff
@@ -129,6 +137,7 @@ func GetRESTHandlers(ctx *context.Provider, opts ...Opt) ([]rest.Handler, error)
 	allHandlers = append(allHandlers, messagingOp.GetRESTHandlers()...)
 	allHandlers = append(allHandlers, routeOp.GetRESTHandlers()...)
 	allHandlers = append(allHandlers, verifiablecmd.GetRESTHandlers()...)
+	allHandlers = append(allHandlers, issuecredentialOp.GetRESTHandlers()...)
```

Comment:
Did you run `make run-openapi-demo` and tested this endpoint in swagger UI?

Reply:
Yeah, I did `make run-openapi-demo`. But, I did not test them. It is covered by unit tests. I checked only the specification. I am going to check it one more time tomorrow (the spec).